src/Equinor.ProCoSys.IPO.Query/GetOutstandingIpos/GetOutstandingIposForCurrentPersonQueryHandler.cs

```diff
@@ -31,34 +32,86 @@ public class GetOutstandingIposForCurrentPersonQueryHandler : IRequestHandler<Ge
             _plantProvider = plantProvider;
         }
 
+        public async Task<List<InvitationDto>> GetNonCanceledInvitations(CancellationToken cancellationToken)
+        {
+
+            var nonCancelledInvitationsFlat =
+                  await (from i in _context.QuerySet<Invitation>()
+                         join p in _context.QuerySet<Participant>() on i.Id equals EF.Property<int>(p, "InvitationId")
+                         where i.Status != IpoStatus.Canceled
+                         select new
+                         {
+                             Id = i.Id,
+                             Description = i.Description,
+                             Status = i.Status,
+
+                             ParticipantId = p.Id,
+                             AzureOid = p.AzureOid,
+                             FunctionalRoleCode = p.FunctionalRoleCode,
+                             Organization = p.Organization,
+                             SignedAtUtc = p.SignedAtUtc,
+                             SortKey = p.SortKey,
+                             Type = p.Type,
+                             SignedBy = p.SignedBy
+
+                         })
+                         .ToListAsync(cancellationToken);
+
+            var nonCancelledInvitationsGrouped = new List<InvitationDto>();
```

Comment:
rename and rearange to this to ease reading
![image](https://user-images.githubusercontent.com/19545528/193745636-5572b960-e5f6-4dd9-b204-756e8e9c12e9.png)